contracts/Identity.sol

```diff
@@ -0,0 +1,166 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import { Hub } from "./Hub.sol";
+import { IERC734Extended } from "./interface/IERC734Extended.sol";
+import { IdentityStorage } from "./storage/IdentityStorage.sol";
+
+contract Identity is IERC734Extended {
+
+    uint256 constant ADMIN_KEY = 1;
+    uint256 constant OPERATIONAL_KEY = 2;
+    uint256 constant ECDSA = 1;
+
+    Hub public hub;
+
+    constructor(address hubAddress) {
+        require(hubAddress != address(0));
+        hub = Hub(hubAddress);
+    }
+
+    modifier onlyContracts(){
+        require(
+            hub.isContract(msg.sender),
+            "Function can only be called by contracts!"
+        );
+        _;
+    }
+
+    function createIdentity(address operational, address admin)
+        public
+        onlyContracts
+        returns (uint72)
+    {
+        require(operational != address(0), "Operational wallet address can't be empty");
+        require(admin != address(0), "Admin wallet address can't be empty");
+        require(operational != admin, "Same address for ADMIN/OPERATIONAL purposes");
```

Comment:
Do you think we really need to be this strict? I think it makes sense to allow the users to use the same key for both op and admin purposes if they like

Reply:
> Do you think we really need to be this strict? I think it makes sense to allow the users to use the same key for both op and admin purposes if they like

I guess it's against initial idea of improved security? And this is something we had previously, but in different form